SQF/dayz_code/actions/AdvancedTrading/functions/z_at_displayBackpackInfo.sqf

```diff
@@ -32,15 +32,17 @@ if (isNumber (configFile >> 'CfgVehicles' >> _class >> 'transportMaxMagazines'))
 	_transportMaxMagazines  = getNumber (configFile >> 'CfgVehicles' >> _class >> 'transportMaxMagazines');
 };
 
+_displayText = "";
+_displayText = getText(configFile >> "CfgVehicles" >> _class >> "descriptionShort");
```

Comment:
DescriptionShort is identical to displayName for all the _DZ backpacks, so in most cases this will be redundant.

For vehicles it will definitely cut off the dialog on normal or larger interface sizes. We already have too much text there. For weapons it will sometimes cut off when the attachment and magazine lists are long like on the M4A1_DZ, etc. The only place it might fit is for magazines, but some descriptions are very long, so I would test on normal, large and very large interface sizes to make sure it doesn't cut off. If you end up using it here are some translations from the A2OA stringtable:
```
<Spanish>Descripción</Spanish>
<Italian>Descrizione</Italian>
<Hungarian>Leírás</Hungarian>
<Polish>Opis</Polish>
<Russian>Описание</Russian>
<German>Beschreibung</German>
<Czech>Popis</Czech>
```

For future reference you don't need to initialize it as ="", since it is in the private array. GetText will always return a string. If there is no entry it will return "" not nil.